samples/csharp/MicrosoftEduGraphSamples/Workflows/AssignmentWorkflow.cs

```diff
@@ -123,5 +123,53 @@ public async Task AssignmentsEvolvableEnumsAsync()
                 Console.WriteLine($"AssignmentsEvolvableEnumsAsync: {ex.ToString()}");
             }
         }
-    }
+
+        /// <summary>
+        /// Workflow to create Assignment resources in resource folder
+        /// </summary>
+        /// <param name="appOnly">True value authenticates the graph client with application permissions only, otherwise it will be created with delegated permissions.</param> 
+        public async Task AssignmentResourceAsync(bool appOnly = false)
+        {
+            try
+            {
+                string assignmentId = string.Empty;
+
+                // Get a Graph client based on the appOnly parameter
+                var graphClient = appOnly ? GraphClient.GetApplicationClient(_config["tenantId"], _config["appId"], _config["secret"]) : GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], _config["teacherAccount"], _config["password"]);
+
+                // Create assignment
+                var assignment = await Assignment.CreateAsync(graphClient, _config["classId"]);
+                assignmentId = assignment.Id;
+                Console.WriteLine($"Assignment created successfully {assignment.Id} in state {assignment.Status}");
+
+                // Set Up Assignment Resources Folder
+                await Assignment.SetupResourcesFolder(graphClient, _config["classId"], assignmentId);
+                Console.WriteLine("SetupResourceFolder creation successful");
+
+                // Adding new Word resource to assignment
+                var requestBody = new EducationAssignmentResource
+                {
+                    DistributeForStudentWork = false,
+                    Resource = new EducationWordResource
+                    {
+                        OdataType = "microsoft.graph.educationWordResource",
+                        DisplayName = "New Word Document.docx",                        
+                    },
+                };
+
+                var resource = await Assignment.PostResourceAsync(graphClient, _config["classId"], assignmentId, requestBody);
+
+                Console.WriteLine($"Resource created successfully {resource.Id} Display Name {resource.Resource.DisplayName}");
```

Comment:
$"Resource created successfully with id : {resource.Id},  Display Name : {resource.Resource.DisplayName}"

Reply:
updated to "$"Resource created successfully with id : {resource.Id}, Display Name : {resource.Resource.DisplayName}"